app/views/media_library/show.html.erb

```diff
@@ -33,11 +33,18 @@
     <div class="row">
       <div class="small-12 columns">
         <div class="title-block">
-          <% breadcrumbs_links = [
+          <% breadcrumbs_links = @media_content.album_id.present? ? [
             {text: 'Media library', link: media_path},
+            {text: 'Album', link: medium_path(@media_content.album_id)},
```

Comment:
Could you try with displaying the album name. You can use truncate to make it smaller

`truncate(@media_content.album.title, length: 20)`